core/commons/che-core-commons-observability/src/test/java/org/eclipse/che/commons/observability/TimedCronExecutorServiceTest.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.commons.observability;
+
+import static org.testng.Assert.assertEquals;
+
+import io.micrometer.core.instrument.MockClock;
+import io.micrometer.core.instrument.Tag;
+import io.micrometer.core.instrument.Tags;
+import io.micrometer.core.instrument.internal.TimedCronExecutorService;
+import io.micrometer.core.instrument.simple.SimpleConfig;
+import io.micrometer.core.instrument.simple.SimpleMeterRegistry;
+import java.text.ParseException;
+import java.util.concurrent.CountDownLatch;
+import org.eclipse.che.commons.schedule.executor.CronExpression;
+import org.eclipse.che.commons.schedule.executor.CronThreadPoolExecutor;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class TimedCronExecutorServiceTest {
```

Comment:
this class should be moved together with `TimedCronExecutorService` to `io.micrometer.core.instrument.internal` package

Reply:
done